src/_components/userPoints/style.ts

```diff
@@ -0,0 +1,33 @@
+import { TouchableOpacity } from 'react-native';
+import styled from 'styled-components/native';
+
+import { theme } from '../../_styles/theme';
+import Typography from '../typography/Typography';
+
+export const UserPointsButton = styled(TouchableOpacity)`
+  padding-top: 4px;
```

Comment:
misschien heb ik er over gekeken maar het coole is dat je nu dit kan doen he: 
`padding: 4px 4px 4px 8px;`

Reply:
zou kunnen dat je dit overal doet :-P